spec/mailers/school_mailer_spec.rb

```diff
@@ -314,14 +314,11 @@
     let(:participant_profile) { create(:ect_participant_profile, training_status: "withdrawn", school_cohort:, user: create(:user, email: "john.clemence@example.com")) }
     let(:sit_profile) { create(:induction_coordinator_profile, schools: [school_cohort.school]) }
     let(:partnership) { create(:partnership) }
-
-    before do
-      create(:induction_record, participant_profile:, partnership:)
-    end
+    let!(:induction_record) { create(:induction_record, participant_profile:, partnership:) }
```

Comment:
just to confirm this is linked to a different school to the profile?

Reply:
Yeah its different by default; I'll make it explicit in the spec though in case that changes in the future